services/QuillLMS/client/app/bundles/Proofreader/components/proofreaderActivities/container.tsx

```diff
@@ -138,9 +138,7 @@ export class PlayProofreaderContainer extends React.Component<PlayProofreaderCon
       const initialPassageData = formatInitialPassage(passage)
       const formattedPassage = initialPassageData.passage
       let currentPassage = formattedPassage
-      if (session.passageFromFirebase && typeof session.passageFromFirebase !== 'string') {
```

Comment:
I think instead of deleting this, you're going to want to do this:

```
      if (session.passageFromFirebase && typeof session.passageFromFirebase !== 'string' && session.passageFromFirebase.length) {
        currentPassage = session.passageFromFirebase
      }
```

Otherwise resuming a session will never work.